<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Production_management</class>
 <widget class="QWidget" name="Production_management">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>966</width>
    <height>458</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Production_management{background-color: rgb(243, 243, 243);}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,9">
   <item>
    <widget class="QWidget" name="panel" native="true">
     <property name="styleSheet">
      <string notr="true">#panel {
background-color: rgb(238, 238, 236);
}

</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QCheckBox" name="checkBox_4">
        <property name="text">
         <string>进度管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_4" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget_4{
background-color: rgb(238, 238, 236);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QCheckBox" name="checkBox">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>进度计划</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#widget{
background-color: rgb(238, 238, 236);
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QPushButton" name="week_plan">
              <property name="text">
               <string>周计划</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="month_plan">
              <property name="text">
               <string>月计划</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBox_2">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>进度填报</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">#widget_2{
background-color: rgb(238, 238, 236);
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QPushButton" name="day_repo">
              <property name="text">
               <string>日报</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="week_repo">
              <property name="text">
               <string>周报</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="month_repo">
              <property name="text">
               <string>月报</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="styleSheet">
            <string notr="true">#widget_5{
background-color: rgb(238, 238, 236);
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="pro_real">
              <property name="styleSheet">
               <string notr="true">/*background-color: #455364;
  border-bottom: 2px solid #455364;*/</string>
              </property>
              <property name="text">
               <string>实际进度</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>台座管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget_3{
background-color: rgb(238, 238, 236);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QPushButton" name="pro_log">
           <property name="text">
            <string>施工日志</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pro_table">
           <property name="text">
            <string>施工表格</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_plan"/>
     <widget class="QWidget" name="page_update"/>
     <widget class="QWidget" name="page_real"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
